Lib/tarfile.py

```diff
@@ -343,7 +343,8 @@ class _Stream:
        _Stream is intended to be used only internally.
     """
 
-    def __init__(self, name, mode, comptype, fileobj, bufsize):
+    def __init__(self, name, mode, comptype, fileobj, bufsize,
+                 compresslevel=9):
```

Comment:
Since `_Stream` is intended to be used only internally, we are free to change the signature of its constructor. The default argument for `compresslevel` is not needed. This just duplicates its value.